### ISSUE TITLE
Add app version to user login API call

### ISSUE THREAD

**Issue by ChasNelson1990** [road86/bahis-desk#47]
### Problem

It's useful to see in the logs which version of the app a user is using for debugging, update roll outs, etc..

### Desired Solution

We could just add an `?version=v1.0.0` tag to the user login request.

**Comment by ChasNelson1990** [comment]
@mixmixmix - it would actually be useful to have this on all calls? The nicest fix for #46 would have been to just handle the old version and new version differently. Thoughts?

**Comment by mixmixmix** [comment]
I agree. As many users will take time to update the app, we have to make sure it is compatible.
